model/user_get.go

```diff
@@ -29,3 +29,14 @@ type UserGetOptions struct {
 	// Page size
 	PerPage int
 }
+
+type UserGetByIdsOptions struct {
```

Comment:
Should we break these structs up instead of having "unusable" fields from a client perspective? e.g. have a `store.UserGetByIdsOptions`?

Reply:
Yeah, that's a good point, although I think `UserGetOptions` might be entirely server-side unlike the new options I'm adding